mcs/class/corlib/Makefile

```diff
@@ -133,8 +133,13 @@ DEFAULT_REFERENCES =
 
 TEST_LIB_REFS = System.Core System
 
+TestModule.dll: ../../../external/corefx/src/System.Runtime/tests/TestModule/System.Reflection.TestModule.il
```

Comment:
This need to use profile specific path, e.g. `$(test_lib_dir)/System.Reflection.TestModule.dll:`